src/FixedDataTableRow.js

```diff
@@ -177,6 +197,11 @@ class FixedDataTableRowImpl extends React.Component {
           {scrollableColumns}
           {columnsLeftShadow}
         </div>
+        {rowExpanded && <div
```

Comment:
In my mind, we'd better keep this wrapping div, as we can have clearly dom structure and if we delete it, we need write more code to position rowExpanded.